test/Test-Automation/docker/docker-compose.yml

```diff
@@ -1,3 +1,4 @@
+version: '3'
```

Comment:
The designation "version" is deprecated.  No longer needed and will throw an error on compile.  Yeah, it was new to me as well.  Just go ahead and delete this line.